src/Pagination/Pagination.scss

```diff
@@ -1,15 +1,214 @@
 @import "variables";
 @import "~bootstrap/scss/pagination";
 
-.page-link {
-    border-color: $pagination-border-color !important;
-    border-radius: 0 !important;
+.pagination {
+  align-items: center;
+
+  .fa {
+    font-size: $pagination-icon-size;
+    line-height: $pagination-icon-size;
+
+    &.fa-angle-left {
+      margin-right: 1rem;
+    }
 
-    div {
-        display: inline-block;
+    &.fa-angle-right {
+      margin-left: 1rem;
     }
+  }
+
+  .dropdown {
+    z-index: 4;
+  }
+}
+
+.page-link {
+  border: none;
+
+  div {
+    display: inline-block;
+  }
 }
 
-.page-item > .btn {
+.page-item {
+  > .btn {
     transition: none;
+    line-height: 1.438rem;
+  }
+}
+
+.pagination-small {
+  .page-link {
+    font-size: $pagination-font-size-sm;
+    line-height: $pagination-link-height;
+    padding: 0 $pagination-padding-y-sm;
+
+    &.previous,
+    &.next {
+      padding: 0 1rem;
+      line-height: $pagination-link-height;
+
+      div {
+        display: flex;
+        align-items: center;
+      }
+    }
+
+    .fa {
+      font-size: $pagination-icon-size-sm;
+      line-height: $pagination-icon-size-sm;
+    }
+  }
+}
+
+
+.pagination-secondary {
+  button.next,
+  button.previous {
+    height: $pagination-secondary-height;
+    padding: 0 1rem;
+
+    .fa {
+      margin: 0;
+      font-size: $pagination-icon-height;
+      line-height: $pagination-icon-line-height;
+    }
+  }
+
+  &.pagination-small {
+        
+    button.next,
+    button.previous {
+      height: $pagination-secondary-height-sm;
+      padding: 0 1rem;
+      line-height: $pagination-link-height;
+
+      .fa {
+        font-size: $pagination-icon-size;
+        line-height: $pagination-icon-size;
+      }
+    }
+  }
+}
+
+.pagination-dark {
+  .pgn__dark-styles {
+    background-color: transparent;
+    color: $white;
+  }
+
+  .page-item {
+    &.disabled .page-link {
+      @extend .pgn__dark-styles;
+    }
+
+    &.active button.page-link {
+      background-color: $pagination-bg;
+      color: $pagination-color;
+    }
+
+    button.page-link {
+      @extend .pgn__dark-styles;
+
+      &:focus {
+        box-shadow: none;
+      }
+    }
+
+    &:not(.active):focus {
+      box-shadow: 0 0 0 .0625rem $pagination-color;
```

Comment:
Would it be possible to use an existing appropriate elevation SCSS variable here, e.g. `$level-1-box-shadow`?

Reply:
fixed